saver.go

```diff
@@ -10,10 +10,15 @@ import (
 )
 
 type SaveQuery struct {
-	Query           string
-	Fields          []string
-	Values          []interface{}
-	PrimaryKeyField *reflect.Value
+	Query  string
+	Fields []string
+	Values []interface{}
+	// Deprecated.
+	// Use PrimaryKeyFields that works fine with composite primary key.
+	// PrimaryKeyField points to the first primary key field in the model. (probably `id` field)
+	PrimaryKeyField    *reflect.Value
```

Comment:
There seems not to be any usage of PrimaryKeyField(s) now, I think they are deletable.

Reply:
Deleted in cc63a81

Shall I also delete primary key checks?

https://github.com/loilo-inc/exql/blob/cc63a81e84c417beac17f4a2f28a9df50c75babd/saver.go#L140-L142